frontend/src/app/modules/invite-user-modal/button/invite-user-button.component.ts

```diff
@@ -18,25 +19,21 @@ export class InviteUserButtonComponent implements OnInit {
   text = {
     button: this.I18n.t('js.invite_user_modal.invite'),
   };
-  canInviteUsersToProject:boolean;
+
+  canInviteUsersToProject$ = this.currentUserService.hasCapabilities$(
+    'memberships/create',
+    this.currentProjectService.id || undefined,
```

Comment:
Checking for capabilities for the current project is probably about as common as checking for global, so this made me wonder whether making hte `CurrentUserService` have access to `CurrentProjectService` is beneficial to reduce needed importing in components

Reply:
That would be fine with me so long as the current project service doesn't end up requiring something in the current user service. In this case, a separate permissions service that depends on the two would probably be easiest.

Until then :+1: for making the API easier for the consuming user